gamestonk_terminal/common/prediction_techniques/regression_model.py

```diff
@@ -2,11 +2,60 @@
 __docformat__ = "numpy"
 from typing import Tuple, List, Any, Union
 import pandas as pd
-from tsxv import splitTrain
+import numpy as np
 from sklearn import linear_model
 from sklearn import pipeline
 from sklearn import preprocessing
 
+# The tsxv dependence was removed so this fails.  Taking from didiers source
```

Comment:
I'm gonna sue you 

![image](https://user-images.githubusercontent.com/25267873/143953812-927234df-acc4-4341-9bdf-5af787d568f8.png)

Reply:
> 

do It u won't